jfr-daemon/src/test/java/com/newrelic/jfr/daemon/app/RemoteEntityBridgeTest.java

```diff
@@ -76,4 +80,28 @@ void awaitRemoteEntityGuid_RemoteAgentBackoffLimit() throws MalformedObjectNameE
     assertEquals(Optional.empty(), target.awaitRemoteEntityId(connection, backoff));
     verify(backoff, times(6)).nextWaitMs();
   }
+
+  @Test
+  void getRemoteEntityName() {
+    doReturn(Optional.of("name")).when(target).getRemoteEntityName(eq(connection));
+    final var remoteEntityName = target.getRemoteEntityName(connection);
```

Comment:
Ugh... the final keyword is so gross in local variables.

Reply:
lol I'll clean these up.